docs/_static/manual_images/supported_features_table.csv

```diff
@@ -0,0 +1,19 @@
+Feature,Minimum Firmware Version,23-XX Speed,23-XX Pulsing,All Other Speed,All Servo

```

Comment:
I'm not bothering trying to hand make this table. We can use CSVs with RST, so I'm going to.